### ISSUE TITLE
What's the equivalent of pgbouncer's server_connect_timeout? 

### ISSUE THREAD

**Issue by naiquevin** [yandex/odyssey#509]
pgbouncer has the `server_connect_timeout` which is documented[1] as follows, 

> server_connect_timeout
If connection and login don’t finish in this amount of time, the connection will be closed. [seconds]
Default: 15.0

Is `pool_timeout` the equivalent setting for the same in odyssey? In case it is, what's the default value of `pool_timeout`. 

[1]: http://www.pgbouncer.org/config.html

**Comment by x4m** [comment]
Yes, pool_timeout is a time that client that wants to get a server connection can wait until reciveing timeout error.
